internal/driver/commands.go

```diff
@@ -129,115 +129,115 @@ var pprofCommands = commands{
 // reported generated by pprof.
 var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
-	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
+	"output": &variable{stringKind, []string{""}, "", helpText("Output filename for file-based outputs")},
```

Comment:
I totally agree with that.
I think we currently allow something like that for non-numeric tags (like you can specify a list of regular expressions the values must match), but I think the sample must match everything in the list to be kept -- and I think you'd suggested matching samples which match any of these tag filters, not all. Would we want to switch to a more or-like convention for this, or stick with it being more like an and?